### ISSUE TITLE
Missing artifacts for 1.83.0.23277 and 1.82.3.23277

### ISSUE THREAD

**Issue by daiyam** [VSCodium/vscodium#1669]
Hello,

Some artifacts are missing from the `1.83.0.23277` and `1.82.3.23277` releases:
- macOS x86
- windows (all)

Same error on both cases:
```
error D:\a\vscodium\vscodium\vscode\node_modules\@parcel\watcher: Command failed.
Exit code: 1
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
Command: node-gyp-build
Arguments: 
Directory: D:\a\vscodium\vscodium\vscode\node_modules\@parcel\watcher
Output:
node:events:495
      throw er; // Unhandled 'error' event
      ^

Error: spawn node-gyp.cmd ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:284:19)
    at onErrorNT (node:internal/child_process:477:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
Emitted 'error' event on ChildProcess instance at:
    at ChildProcess._handle.onexit (node:internal/child_process:290:12)
    at onErrorNT (node:internal/child_process:477:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
  errno: -4058,
  code: 'ENOENT',
  syscall: 'spawn node-gyp.cmd',
  path: 'node-gyp.cmd',
  spawnargs: [ 'rebuild' ]
}
```

https://github.com/VSCodium/vscodium/actions/runs/6408642071/job/17398088692
https://github.com/VSCodium/vscodium/actions/runs/6409117121/job/17399626498

On local macOS x86, another dependency doesn't want to build.
On local windows, no issues.

**Comment by leefordjudes** [comment]
Kindly add binaries (1.83) for windows

**Comment by daiyam** [comment]
Sorry, I don't have time, busy in RL.

**Comment by daiyam** [comment]
I've tested few things but nothing work out. I've tested with the latest Windows image from GitHub, still the same.

**Comment by sid-the-sloth** [comment]
Please add back the AppImage build.

**Comment by daiyam** [comment]
@sid-the-sloth see #1654...

**Comment by daiyam** [comment]
Oh shit! The issue is due to node `v18.18`! Using `v18.17` seems to fix the issue! #1689 (finally done a PR after all the shit I've done in the master...)

**Comment by daiyam** [comment]
`1.83.0.23277` seems to be complete!

**Comment by daiyam** [comment]
I will leave `1.82.3.23277` as it is.

**Comment by syndenbock** [comment]
Unfortunately VSCodium (win32-x64) no longer launches for me on this release. 
I'm not sure if it's related to the new build settings and I don't have VS Code installed so I didn't open a separate issue.

**Comment by daiyam** [comment]
@syndenbock It's not only you!

- windows x64: broken
- macos x64: ok

**Comment by daiyam** [comment]
Here the error `Loading "@vscode/policy-watcher" failed`

`@vscode/sqlite3` and `@parcel/watcher` aren't there either. And maybe others. I will check that out tomorrow.

**Comment by daiyam** [comment]
On local windows:
- node `v18.18`: `.node` are built
- node `v18.17`: `.node` are **not** built

No errors for either versions...

This is why the current issue is "fixed" with `v18.17`...
Weirdly macOS isn't affected by this problem. ???

**Comment by daiyam** [comment]
I'm woking on a fix but for the Insiders version (#1690). When validated, I will make it for the regular one.

**Comment by daiyam** [comment]
The Insiders versions are out and are working correctly.

**Comment by daiyam** [comment]
I'm re-releasing the fixed version as `1.83.0.23283`. It should out in about an hour.

**Comment by daiyam** [comment]
There were 2 issues:
- node `v18.18` generate errors on Windows and macOS x64
- on Windows, bad config of custom node-gyp for the spectre mitigation patch (the way to configure have changed)

**Comment by daiyam** [comment]
Validated, Windows x64 and macOS x64 of `1.83.0.23283` are launching fine!

**Comment by peter-fb** [comment]
I just upgraded, Windows 10 x64, it seems to work fine. Thank you! :)

**Comment by syndenbock** [comment]
> Oh shit! The issue is due to node `v18.18`! Using `v18.17` seems to fix the issue! #1689 (finally done a PR after all the shit I've done in the master...)

A new LTS version 18.18.1 of node was released 5 hours ago including a fix for node images giving an error. 
Could this have been the issue here so it works with 18.18.x again?
